spec/views/days_view_spec.exs

```diff
@@ -0,0 +1,26 @@
+defmodule Janitor.DaysViewsSpec do
+  use ESpec.Phoenix, view: Janitor.DaysView
+  alias Janitor.DayFactory
+  alias Janitor.Day
+
+  def render_days(days) do 
+    days |> Enum.map(
+      fn (day) -> 
```

Comment:
I think it would be a bit cleaner if this was factored out as a private function of this module and given a good, explanatory name.